### PR TITLE
Add featured collection cards

### PR DESCRIPTION
**Pull request by wulman16**
- Content for Explore Our Collections cards is populated using a presenter
- Tile styles from Emory pattern library are selected and adapted for Lux
- Position of facets is adjusted to match position of collection cards
- Script is added to dynamically adjust the height of collection cards to match the height of the facets panel in desktop view
- Links to the featured collections currently direct to the homepage

### CODE REVIEW

**Comment by wulman16** [comment]
Screen Shot 2020-04-09 at 10 10 56 AM

File: app/assets/javascripts/explore_collections.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  } else {
+    setDefaultHeight()
+  }
+}
+
+function desktopWidth(w) {
+  return w.matchMedia("(min-width: 992px)").matches
+}
+
+function matchFacetsHeight() {
```
Comment (bwatson78) [inline app/assets/javascripts/explore_collections.js:18]: This cuts off the text of the card and doesn't allow user to scroll to see the rest. If that's the intended behavior, that's fine. If not, then something like overflow scroll needs to be implemented.

Screen Shot 2020-04-09 at 11 04 18 AM

Comment (wulman16) [reply app/assets/javascripts/explore_collections.js:18]: You may not have as many facets in your local instance as we do in production. As a safeguard, I added a check to make sure that the height of the facets panel is at least as tall as the minimum card height before changing the card height.

File: app/views/catalog/_explore_collections.html.erb [hunk 2]
```diff
@@ -0,0 +1,18 @@
+<% presenter = ExploreCollectionsPresenter.new %>
+<% collections = presenter.collections %>
+<div class="facets-header">
+  <h2 class="facets-heading">Explore Our Collections</h2>
```
[the remark this answers is not in the document]
Comment (wulman16) [reply app/views/catalog/_explore_collections.html.erb]: I just changed the heading to be our `$base-blue`

**Comment by wulman16** [comment]
Screen Shot 2020-04-09 at 12 32 09 PM

**Pull request merged**